### PR TITLE
quickstart guides

### PR DESCRIPTION
**Pull request by adean20csc**
This is the initial PR for the quickstart guides. As each guide is submitted there is code cleanup that has to be done so they will be submitted as they are worked.

First is the quickstart guide, which is a reference to the others.
The first quickstart guide is the autodeploynode.

The next PR will be the autodeploynode cleanup.

### CODE REVIEW

File: docsite/source/autodeploynode_quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
+Autodeploynode Quick Start Guide
+================================
+
+This Quick Start Guide describes how to use CSC DCAF Automation to provision and
+configure the Autodeploynode. It uses the following resources:
+
+- `**dcaf/autodeploynode** <https://github.com/csc/dcaf/autodeploynode>`_ - To
```
Comment (mtnbikenc) [inline docsite/source/autodeploynode_quickstart.rst]: Nested markup does not work, so these links should not be marked as bold (using **).

File: docsite/source/quickstart.rst [hunk 2]
```diff
@@ -1,182 +1,25 @@
 CSC DCAF Automation Quick Start Guide
 =====================================
 
-This quick start guide describes how to use the following Automation projects:
+This quick start guide contains links to the various CSC DCAF module quickstart
+guides. Refer to each as needed.
 
-- **CSC DCAF Modules**
+  `AutoDeployNode Module <http://csc.github.io/dcaf/autodeploynode_quickstart.html>`_
```
Comment (mtnbikenc) [inline docsite/source/quickstart.rst]: The double-space indent for these lines makes the text look like a 'quote'.

File: docsite/source/quickstart.rst [hunk 3]
```diff
@@ -1,182 +1,25 @@
...
+  Used to provision and configure the AutoDeployNode.
 
-  - **AutoDeployNode** Provisions a CSC DCAF AutoDeployNode which is used as the
-    basis for all DCAF automation. The AutoDeployNode contains all the
-    automation resources and is used to perform all automation tasks.
-  - **Bare-Metal-OS** - Provides an automated bare-metal deployment of the
-    Red Hat Enterprise Linux OS using Hanlon.
+  `Bare-metal-OS Module <http://csc.github.io/dcaf/bare_metal_os_quickstart.html>`_
```
Comment (mtnbikenc) [inline docsite/source/quickstart.rst]: I suggest using Bare-Metal-OS when referring to the module.

File: docsite/source/quickstart.rst [hunk 4]
```diff
@@ -1,182 +1,25 @@
...
+  Used to  provision an OS on bare-metal hardware.
 
-- **Hanlon** - Hanlon is an advanced provisioning platform which can provision
-  both bare-metal and virtual systems.
-- **Slimer** - Slimer is a fork of abrezhnev/slimer to deploy the Red Hat
-  OpenStack Platform with high availability DCAF provisioned RHEL
+  `Rhel-osp Module <http://csc.github.io/dcaf/rhel_osp_quickstart.html>`_ Used
```
Comment (mtnbikenc) [inline docsite/source/quickstart.rst]: I suggest using RHEL-OSP.

File: docsite/source/quickstart.rst [hunk 5]
```diff
@@ -1,182 +1,25 @@
...
+  to deploy the Red Hat OpenStack Platform with high availability on base RHEL OS
   installations.
-- **Ansible-ScaleIO** - Ansible-ScaleIO is a fork of sperreault/ansible-scaleio
-  to install, configure and manage ScaleIO. ScaleIO provides additional storage
-  capabilities to the Red Hat OpenStack Platform.
 
-Before You Begin
-----------------
+  `Kvm-host Module <http://csc.github.io/dcaf/kvm_host_quickstart.html>`_ Used
```
Comment (mtnbikenc) [inline docsite/source/quickstart.rst]: I suggest using KVM-Host.

**Comment by mtnbikenc** [comment]
Doc review :+1:

**Comment by jcpowermac** [comment]
Review 👍

**Pull request merged**